package-lock.json

```diff
@@ -78,6 +78,150 @@
         "@nomicfoundation/slang-win32-x64-msvc": "0.13.1"
       }
     },
+    "crates/solidity/outputs/npm/package/target/npm/main/node_modules/@nomicfoundation/slang-darwin-arm64": {
```

Comment:
I think this might have been added during rebase?

Reply:
Done